### PR TITLE
Add check_tidy.py option to edit the detected clang format errors.

### PR DESCRIPTION
**Pull request by hs0225**
- Add  `autoedit` option that edits to files instead of displaying a diff.(only clang-format)

IoT.js-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

File: tools/check_tidy.py [hunk 1]
```diff
@@ -28,6 +30,16 @@
 from common_py.system.executor import Executor as ex
 
 
+def parse_option():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--autoedit', action='store_true', default=False,
+        help='edits to files instead of displaying a diff' +
```
Comment (galpeter) [inline tools/check_tidy.py]: There is no need for the `+` sign. The concatenation will also happen without it (strings are automatically concatenated).

Comment (galpeter) [inline tools/check_tidy.py]: Maybe a bit better help message would be: `Automatically edit/fix the detected clang format problems. No diff(s) will be created/displayed.`

Comment (hs0225) [inline tools/check_tidy.py]: @galpeter Done. Thanks for your review.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**